index.html

```diff
@@ -1930,6 +1925,43 @@ <h3>
           </li>
         </ol>
       </section>
+      <section>
+        <h3>
+          <code>splashscreens</code> member
+        </h3>
+        <p>
+          A <dfn>splash screen</dfn> is a graphical representation (usually an
```

Comment:
should this read `splashscreens` instead of `splash screen`?

the other member sections start [like this](https://github.com/w3c/manifest/pull/530/files#diff-eacf331f0ffc35d4b482f1d15a887d3bR1945). is that an issue